internal/executors/http/basic_authorization_header.go

```diff
@@ -7,18 +7,19 @@ import (
 
 type basicAuthorizationHeader struct {
 	username string
-	password string
+	password []byte
 }
 
 func NewBasicAuthorizationHeader(u string, p string) AuthorizationHeaderGenerator {
 	return &basicAuthorizationHeader{
 		username: u,
-		password: p,
+		password: []byte(p),
```

Comment:
It is already a `string` in `HttpRequestParameters` and in `NewIasAuthorizationHeader` it seems to be extracted from a json that is also a string.  So it is already living in the memory as a string immutable. 

Maybe we should consider doing the conversion to `[]byte` at the topmost part, in order to avoid having the password elsewhere and only copying it into another slice at the bottom.

So in my opinion as it is right now, it is a bit redundant.

Reply:
Reverted.